File: beginner/1160/1160.cpp
#include <iostream>
using namespace std;

int main() {
    int T;
    cin >> T;
    while (T--) {
        int population_A, population_B;
        float growing_rate_A, growing_rate_B;
        cin >> population_A >> population_B >> growing_rate_A >> growing_rate_B;

        int time = 0;
        while (time <= 100 && population_A <= population_B) {
            population_A += (int)(population_A * growing_rate_A / 100);
            population_B += (int)(population_B * growing_rate_B / 100);
            time++;
        }

        if (time > 100) cout << "Mais de 1 seculo.\n";
        else cout << time << " anos.\n";
    }
    return 0;
}

File: beginner/1160/1160.lua
local T = tonumber(io.read())

for i = 1, T do
    local A, B, TA, TB = io.read():match("(%S+) (%S+) (%S+) (%S+)")
    A, B, TA, TB = tonumber(A), tonumber(B), tonumber(TA), tonumber(TB)

    local time = 0
    while time <= 100 and A <= B do
        A = A + math.floor((A * TA) / 100)
        B = B + math.floor((B * TB) / 100)
        time = time + 1
    end

    if time > 100 then
        print("Mais de 1 seculo.")
    else
        print(string.format("%d anos.", time))
    end
end